lib/Remote/RemoteWebDriver.php

```diff
@@ -519,11 +552,17 @@ public function getCapabilities()
      *
      * @param string $selenium_server_url The url of the remote Selenium WebDriver server
      * @param int $timeout_in_ms
+     * @param bool $w3c_compliant
      * @return array
      */
     public static function getAllSessions($selenium_server_url = 'http://localhost:4444/wd/hub', $timeout_in_ms = 30000)
     {
-        $executor = new HttpCommandExecutor($selenium_server_url);
+        // BC layer to not break the method signature
+        if (func_num_args() > 2 && func_get_arg(2)) {
```

Comment:
Well, forcing people to add a new argument just to get an error looks weird to me.

Reply:
I agree, but what do you suggest? Catching the exception and re-throwing another one? Letting the original exception bubbling?